src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Bookstore from './components/Bookstore';
+// import { Provider } from 'react-redux';
```

Comment:
- [ ] Please kindly remove commented code code from your code base as it makes your code dirty and less readable.

Reply:
done